File: 2000 98 & ME/QuickStart/howto/samples/Services/Timers/timer/timer.cs
using System;
using System.Threading;
using System.Timers;
	
public class WaitableTimerSample
{
    public static void Main(String[] args) 
	{
		System.Timers.Timer aTimer = new System.Timers.Timer();
	
		aTimer.Tick += new EventHandler(OnTimer);

		aTimer.Interval = 1000;
		aTimer.Enabled = true;
		
		Console.WriteLine("Press \'q\' to quit the sample");	
		while(Console.Read()!='q');
    }

	public static void OnTimer(Object source, EventArgs e)
	{
		Console.WriteLine("Hello World!");
	}
}

File: 2000 98 & ME/QuickStart/howto/samples/Services/Timers/timer/VB/Timer.vb
Imports System
Imports System.Threading
Imports System.Timers
	
Namespace Timer

    Module Timer

        Public Sub Main()
            Dim aTimer As System.Timers.Timer
            aTimer = New System.Timers.Timer
	
            AddHandler aTimer.Tick, AddressOf OnTimer

            aTimer.Interval = 1000
            aTimer.Enabled = True
		
            Console.WriteLine("Press 'q' to quit the sample")
            While (Console.Read() <> 113)
            End While
        End Sub

        Public Sub OnTimer(ByVal source As Object, ByVal e As EventArgs)
            Console.WriteLine("Hello World!")
        End Sub

    End Module

End Namespace
